WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -221,7 +235,6 @@ nick_the_ever_craving = {
 
 nick_the_anger_filled = {
 	allow = {
-		religion = buddhist
 		OR = {
```

Comment:
The other religions have or will get other specific nicknames for their anger. It gives flavour specifically to them.